pkg/core/auth/authinfo/pq/store.go

```diff
@@ -94,10 +98,9 @@ func (s AuthInfoStore) CreateAuth(authinfo *authinfo.AuthInfo) (err error) {
 		return skydb.ErrUserDuplicated
 	}
 
-	// TODO:
-	// if err := s.UpdateUserRoles(authinfo); err != nil {
-	// 	return skydb.ErrRoleUpdatesFailed
-	// }
+	if err := s.updateUserRoles(authinfo); err != nil {
+		return skydb.ErrRoleUpdatesFailed
+	}
```

Comment:
Please update the signup handler to assign default roles to authinfo when calling CreateAuth.